zcash-issue-dag.py

```diff
@@ -308,7 +330,19 @@ def main():
             dg.remove_nodes_from(nx.compose_all(ignore))
 
     # Prune nodes that are not downstream of any open issues.
-    if PRUNE_FINISHED:
+    if PRUNE_FINISHED == 'targets':
+        closed_targets = [n for n in dg.nodes if n.is_target and n.state == 'closed']
+        for target in closed_targets:
+            try:
+                ancestors = nx.ancestors(dg, target)
+                if not any(n.state != 'closed' for n in ancestors):
```

Comment:
```suggestion
                if all(n.state == 'closed' for n in ancestors):
```

Reply:
(Incidentally, TIL you can omit the parens around a generator expression [if it's the only argument to a function](https://docs.python.org/3/reference/expressions.html#generator-expressions).)